src/AspNet.Security.OAuth.Shopify/ShopifyAuthenticationHandler.cs

```diff
@@ -28,72 +29,72 @@ public class ShopifyAuthenticationHandler : OAuthHandler<ShopifyAuthenticationOp
             [NotNull] OAuthTokenResponse tokens)
         {
-            var uri = string.Format(ShopifyAuthenticationDefaults.FormatUserInformationEndpoint,
-                properties.Items[ShopifyAuthenticationDefaults.ShopNameAuthenticationProperty]);
+            string uri = string.Format(CultureInfo.InvariantCulture, ShopifyAuthenticationDefaults.UserInformationEndpointFormat, properties.Items[ShopifyAuthenticationDefaults.ShopNameAuthenticationProperty]);
 
-            var request = new HttpRequestMessage(HttpMethod.Get, uri);
+            HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, uri);
             request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
             request.Headers.Add("X-Shopify-Access-Token", tokens.AccessToken);
 
-            var response = await Backchannel.SendAsync(request, HttpCompletionOption.ResponseHeadersRead, Context.RequestAborted);
+            HttpResponseMessage response = await Backchannel.SendAsync(request, HttpCompletionOption.ResponseHeadersRead, Context.RequestAborted);
             if (!response.IsSuccessStatusCode)
             {
                 Logger.LogError("An error occurred while retrieving the user profile: the remote server " +
                                 "returned a {Status} response with the following payload: {Headers} {Body}.",
                                 /* Status: */ response.StatusCode,
                                 /* Headers: */ response.Headers.ToString(),
                                 /* Body: */ await response.Content.ReadAsStringAsync());
-            
+
                 throw new HttpRequestException("An error occurred while retrieving the shop profile.");
             }
 
-            var payload = JObject.Parse(await response.Content.ReadAsStringAsync());
+            JObject payload = JObject.Parse(await response.Content.ReadAsStringAsync());
 
             // In Shopify, the customer can modify the scope given to the app. Apps should verify
             // that the customer is allowing the required scope.
-            var actualScope = tokens.Response["scope"].ToString();
-            var isPersistent = true;
+            string actualScope = tokens.Response["scope"].ToString();
+            bool isPersistent = true;
 
             // If the request was for a "per-user" (i.e. no offline access)
-            if (tokens.Response.TryGetValue("expires_in", out var expireInValue))
+            if (tokens.Response.TryGetValue("expires_in", out JToken expireInValue))
             {
                 isPersistent = false;
 
-                var expires = DateTimeOffset.UtcNow.AddSeconds(Convert.ToInt32(expireInValue.ToString()));
-                identity.AddClaim(new Claim(ClaimTypes.Expiration, expires.ToString("O", CultureInfo.InvariantCulture), ClaimValueTypes.DateTime));
+                if (Int32.TryParse(expireInValue.ToString(), out int expireIn))
```

Comment:
Use the overload that accepts `IFormatProvider` and specify `CultureInfo.InvariantCulture`.